app/utils/callback/lava.py

```diff
@@ -594,12 +448,194 @@ def _translate_log_end_lines(results, translate_map):
                 result['log_end_line'] = new_log_end_line
 
 
-def _prepare_log(log):
-    log = yaml.load(log, Loader=yaml.CLoader)
-    for i, log_line in enumerate(log, 1):
-        log_line['msg'] = unicode(log_line['msg'])
-        log_line['org_num'] = i
-    return log
+class LavaCallback(object):
```

Comment:
Maybe a docstring for the class or the `__init__` constructor would be good, but that's not a blocking issue.

Reply:
This commit also has no description.

Reply:
added docstrings